pom.xml

```diff
@@ -95,7 +95,7 @@
         <version.org.codehaus.woodstox.woodstox-core>5.0.3</version.org.codehaus.woodstox.woodstox-core>
         <version.org.fusesource.jansi>1.11</version.org.fusesource.jansi>
         <version.org.jboss.aesh>0.66.11</version.org.jboss.aesh>
-        <version.org.jboss.byteman>3.0.3</version.org.jboss.byteman>
+        <version.org.jboss.byteman>4.0.0-ALPHA</version.org.jboss.byteman>
```

Comment:
@ropalka WildFly Core 3.0.0.CR1 is probably going to be out early next year and can't have dependencies that are not .Final (or at least .CR with a clear schedule to be .Final in time for core's .Final)

I think this is going to be a general issue with this JDK 9 work. Our schedule requires releases before JDK 9 so any components that are making changes that target 9 may not be ready in time.